x-pack/plugins/fleet/server/services/package_policy.ts

```diff
@@ -1171,7 +1169,110 @@ export function overridePackageInputs(
   return resultingPackagePolicy;
 }
 
-function deepMergeVars(original: any, override: any): any {
+export function preconfigurePackageInputs(
```

Comment:
could these new functions be moved to separate files? this service is more than 1000 lines long.

Reply:
Yes I think it will make sense to refactorize this whole service, I am going to create a follow up issue to that if it's correct for you?

Reply:
I created that issue here https://github.com/elastic/kibana/issues/119844

Reply:
thanks!